menu/src/main/java/com/jagrosh/jdautilities/menu/Menu.java

```diff
@@ -141,7 +139,7 @@ protected boolean isValidUser(User user)
      *
      * @return {@code true} if the User is valid, {@code false} otherwise.
      */
-    protected boolean isValidUser(User user, @Nullable Guild guild)
+    protected boolean isValidUser(User user, Guild guild)
```

Comment:
Why should we remove an annotation like this?
This method is not explicitly internal so it should be properly marked as nullable